src/ShallowWrapper.js

```diff
@@ -63,12 +63,24 @@ function filterWhereUnwrapped(wrapper, predicate) {
   return wrapper.wrap(compact(wrapper.getNodes().filter(predicate)));
 }
 
+function validateOptions(options) {
+  if (
+    options.lifecycleExperimental != null &&
+    options.lifecycleExperimental === options.disableLifecycleMethods
```

Comment:
i think this comparison should probably just `‼` both options, and throw if they're the same, as long as *both* of them aren't `== null`?